pubspec.yaml

```diff
@@ -54,10 +53,31 @@ dependency_overrides:
     git:
       url: https://github.com/incrediblezayed/file_saver.git
       ref: main
+  at_contacts_flutter:
```

Comment:
Can we use hosted version isntead of git reference ?

Reply:
I am not sure which version must be used for the rest of the packages except onboarding, so I left as it was.

Reply:
I have removed the dependency overrides and used the hosted ones